src/facade/redis_parser.h

```diff
@@ -98,8 +100,8 @@ class RedisParser {
   absl::InlinedVector<std::pair<uint32_t, RespVec*>, 4> parse_stack_;
   std::vector<std::unique_ptr<RespVec>> stash_;
 
-  using BlobPtr = std::unique_ptr<uint8_t[]>;
-  std::vector<BlobPtr> buf_stash_;
+  using Blob = std::vector<uint8_t>;
```

Comment:
can you extract this change as well to a new PR?

Reply:
it is part of this PR though, because that's the only way we can do tracking (we have no way of getting the array size)